web/react/components/create_post.jsx

```diff
@@ -292,6 +294,24 @@ export default class CreatePost extends React.Component {
         const draft = PostStore.getDraft(channelId);
         return draft.previews.length + draft.uploadsInProgress.length;
     }
+    handleArrowUp(e) {
+        if (e.keyCode === KeyCodes.UP && this.state.messageText === '') {
+            e.preventDefault();
+
+            const channelId = ChannelStore.getCurrentId();
+            const lastPost = PostStore.getCurrentUsersLatestPost(channelId);
+            var type = (lastPost.root_id && lastPost.root_id.length > 0) ? 'Comment' : 'Post';
+
+            AppDispatcher.handleViewAction({
+                type: ActionTypes.RECIEVED_EDIT_POST,
+                refoucsId: '#post_textbox',
```

Comment:
I guess you mean ````RECIEVED_EDIT_POST````?
"RECEIVED" has this typo in all jsx files, I used this spelling to be in line with the other event names.
Should I fix this and/or the other typos or submit an additional merge request?